crmsh/config.py

```diff
@@ -277,7 +277,7 @@ def get(self, value):
         'from_time': opt_string('-12H'),
         'compress': opt_boolean('yes'),
         'speed_up': opt_boolean('no'),
-        'collect_extra_logs': opt_string('/var/log/messages /var/log/pacemaker/pacemaker.log /var/log/pacemaker.log /var/log/ha-cluster-bootstrap.log'),
+        'collect_extra_logs': opt_string('/var/log/messages /var/log/pacemaker/pacemaker.log /var/log/pacemaker.log /var/log/crmsh/ha-cluster-bootstrap.log'),
```

Comment:
IIUC, this should let `crm report` correctly collect this log file?

Reply:
Not sure. I saw no difference in the hb report.

Reply:
The point is, as long as it's collected into a hb_report tar ball :-)

Reply:
I think, it is simply ignored. (see https://github.com/ClusterLabs/crmsh/blob/master/hb_report/hb_report.in#L64)

Reply:
@liangxin1300 , could you please approve or comment on this change?

Reply:
> I think, it is simply ignored. (see https://github.com/ClusterLabs/crmsh/blob/master/hb_report/hb_report.in#L64)

Simply ignored or collected? We definitely want the latter :-)

Reply:
> > I think, it is simply ignored. (see https://github.com/ClusterLabs/crmsh/blob/master/hb_report/hb_report.in#L64)
> 
> Simply ignored or collected? We definitely want the latter :-)

The second. It exists in the constants.EXTRA_LOGS, but the hb_report doesn't care about the value.

Reply:
@liangxin1300, it sounds like `crm report` no longer collects bootstrap log file even without @aleksei-burlakov's change in there?

Reply:
@gao-yan, I've tested @aleksei-burlakov's patch, `hb_report` will collect bootstrap log file which located at `/var/log/crmsh`